context/app/static/js/shared-styles/sections/LabelledSectionText/LabelledSectionText.jsx

```diff
@@ -38,12 +39,14 @@ LabelledSectionText.propTypes = {
   iconTooltipText: PropTypes.string,
   bottomSpacing: PropTypes.number,
   childContainerComponent: PropTypes.elementType,
+  fontWeight: PropTypes.number,
 };
 
 LabelledSectionText.defaultProps = {
   iconTooltipText: undefined,
   bottomSpacing: undefined,
   childContainerComponent: undefined,
+  fontWeight: undefined,
```

Comment:
This should match the default value defined in line 16
```suggestion
  fontWeight: 400,
```

Reply:
As mentioned before, I was working on the wrong branch! I reverted my changes and pushed. I will make sure to still apply your comments on the branch.

Reply:
Reference the theme?